projects/lib/src/board/westernboard.h

```diff
@@ -76,6 +76,29 @@ class LIB_EXPORT WesternBoard : public Board
 		/*! Movement mask for Rook moves. */
 		static const unsigned RookMovement = 8;
 
+		/*! Types of Pawn moves. */
+		enum StepType: const char
```

Comment:
Good point - const char is a remnant of an idea I had in the start but I did not pursue. Deleted.
StepTypes: good idea - adopted.